crates/common/consensus/lean/src/attestation.rs

```diff
@@ -0,0 +1,19 @@
+use alloy_primitives::B8;
+use serde::{Deserialize, Serialize};
+use ssz_derive::{Decode, Encode};
+use ssz_types::{BitList, VariableList, typenum::U4096};
+use tree_hash_derive::TreeHash;
+
+use crate::vote::Vote;
+
+/// Represents an attestation in the Lean chain.
+///
+/// See the [Lean specification](https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md#attestation)
+/// for detailed protocol information.
+#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash)]
+pub struct Attestation {
+    // U4096 = VALIDATOR_REGISTRY_LIMIT
```

Comment:
```suggestion
    /// U4096 = VALIDATOR_REGISTRY_LIMIT
```

this should be a doc comment

Reply:
Updated in [c964b3b](https://github.com/ReamLabs/ream/pull/755/commits/c964b3b257e7e39fa65dc937800dc705d81e3f84)